enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+status: implementable
+see-also:
+  - [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md)
+  - [custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43)
+  - [Enhancement for validating bundles in a catsrc image #48](https://github.com/operator-framework/enhancements/pull/48)
+  - [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub]
+---
+
+# Audit command operation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+1. Should this command to be added to SDK or OPM or in a new tool?
+
+**Context**
+
+The audit command will perform checks that are implemented and are provided via SDK tool commands `bundle validate` and `scorecard` already. Then, it could be done from SDK binary or use it. However, the OPM tool is the tool that has been used with the purpose of admin the catalogs. 
+
+Note that SDK will also get starts to allow users to work with catalogs by using opm. See the proposal [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md).
+
+Also, the motivations of the proposal [Custom operator validation](https://github.com/operator-framework/enhancements/pull/43)  confirms the interest of OPM users in the audit command. 
+
+The downside of create a new tool for this purpose shows mainly be the effort to keep it maintained and not take advantage of is  implemented already as ask for users which who requires to audit the OLM index catalogs and would like to use this feature also install an additional tool since they probably work already with opm or sdk and maybe both indeed. 
+
+**Conclusion**
+//TODO: Add the consensus reached out. 
+
+
+## Summary
+
+This proposal describes a new operation provided via an alpha command/solution which has the purpose to audit all bundles of some specific OLM index catalog and then, output a report.
+
+## Motivation
+
+### Goals
+
+- Be able to audit all bundles of an OLM index catalog and output a report
+- Be able to easily check and identify aspects of the operators bundles which will be published on it such as:
+    - check if they are using deprecated apis
+    - check if they are built with sdk and its versions
+    - the language used 
+    - check if they are multi-arch
+    - check if they are using some relevant feature provided by looking their annotations
+    - check if they are following the recommendations and conventions defined
+    - check all warnings and errors which will be raised by using the SDK features scorecard and bundle validate
+- Be able to extract a report with the audit results and in some formats such as json. 
+- Be able to run specific custom checks for each bundle in order to audit the catalog
+- Provide a documentation over how to use the audit command
```

Comment:
```suggestion
- Provide documentation on how to use the audit command
```

Reply:
These goals all sound like the items that a Catalog maintainer would want to know NOT an operator developer.